### PR TITLE
Partial (2/3) spanish translation ch10-02-traits.md

### PR DESCRIPTION
**Pull request by jossue33**
Number of words: 1037
Line: 189-405

### CODE REVIEW

File: second-edition/src/ch10-02-traits.md [hunk 1]
```diff
@@ -281,59 +281,59 @@ let tweet = Tweet {
...
-### Trait Bounds
+### Los límites de los rasgos
 
-Now that we’ve defined traits and implemented those traits on types, we can use
-traits with generic type parameters. We can constrain generic types so that
-rather than being any type, the compiler will ensure that the type will be
-limited to those types that implement a particular trait and thus have the
-behavior that we need the types to have. This is called specifying *trait
-bounds* on a generic type.
+Ahora que hemos definidos los ragos y los hemos implementado en los tipos, podemos usar
```
Comment (thecodix) [inline second-edition/src/ch10-02-traits.md]: rasgos

**Comment by jossue33** [comment]
@thecodix todo listo.

**Comment by thecodix** [comment]
Closed cause contents are just copied from google translate

**Comment by thecodix** [comment]
machine translated

**Pull request closed without merging**